furusiya/model/components/base.py

```diff
@@ -0,0 +1,17 @@
+class Component:
```

Comment:
Same comment here, please call it `AbstractComponent` or `BaseComponent` or something.